lib/Yam/ControlCenter.pm

```diff
@@ -0,0 +1,374 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles Control Center
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Yam::ControlCenter;
+use parent 'YaST::PageBase';
```

Comment:
let's duplicate this page for now under Yam, I would like to not have dependencies with previous code, so it is easier to follow for everyone, so let's create Yam::PageBase to inherit from.